Pod/Classes/VOKCoreDataManager.m

```diff
@@ -524,6 +583,35 @@ - (NSManagedObjectContext *)temporaryContext
     return [self tempManagedObjectContext];
 }
 
+- (NSManagedObjectContext *)temporaryInMemoryContext
+{
+    //Spin up a new context
+    NSManagedObjectContext *context;
+    context = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
+    
+    //Use the same MOM as the main context.
+    NSManagedObjectModel *model = self.managedObjectModel;
+    
+    //Use an in-memory store
+    NSPersistentStoreCoordinator *coordinator;
+    coordinator = [[NSPersistentStoreCoordinator alloc] initWithManagedObjectModel:model];
```

Comment:
This is personal preference, but I'm not a fan of using two lines to define and then set variables like this (and `context`)